### DISCUSSION TITLE
Machine Account

### DISCUSSION THREAD

**Discussion by ShaharMachine** [community/community#131874] [New to GitHub]
So i readed through the github policy about allowing a second free account if its a machine account i wanted to post some static web pages on github pages not on my main account so there is seperation between hosting websites and coding is this solution fits the policy i am not so sure...

**Comment by paiineli** [comment]
Hi,

It’s great that you’re looking to understand GitHub's policy regarding multiple accounts. According to GitHub’s guidelines, creating a second account for specific purposes like hosting static web pages is generally acceptable, especially if it's for a machine or organizational use.

However, to ensure compliance with their policies, here are a few points to consider:

Purpose of the Account: Make sure that the second account is solely for hosting and does not overlap with your main account's usage.

Separate Projects: Keep the projects clearly separated between your main account and the secondary account to avoid any confusion.

Documentation: Familiarize yourself with GitHub’s terms regarding multiple accounts and ensure that your use case fits their criteria.

If you’re still unsure, I recommend reaching out to GitHub Support for clarification specific to your situation. They can provide the most accurate guidance.

Best,
